vendor/bootstrap/js/bootstrap.js

```diff
@@ -756,7 +756,8 @@
       selector = selector && /#/.test(selector) && selector.replace(/.*(?=#[^\s]*$)/, '') //strip for ie7
     }
 
-    $parent = selector && $(selector)
+    // Removed for jQuery 3.5.1
```

Comment:
Nitpick: commented code

Reply:
This is intentional and the only change needed (for now) to make the Bootstrap components work with jQuery 3.